ansible/roles/isic/meta/main.yml

```diff
@@ -4,7 +4,7 @@ dependencies:
   - role: girder
   - role: supervisor
   - role: worker
-  - role: large-image
+  - role: large_image
```

Comment:
Any reason for this change other than preference?

Reply:
Consistency with all the other naming of the large_image plugin, package, and repo.